content/pt/docs/reference/access-authn-authz/bootstrap-tokens.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Autenticando com Tokens de Inicialização
+content_type: concept
+weight: 20
+---
+ 
+<!-- overview -->
+ 
+{{< feature-state for_k8s_version="v1.18" state="stable" >}}
+ 
+Os tokens de inicialização são um _bearer token_ simples que devem ser utilizados
+ao criar novos clusters ou para quando novos nós são registrados a clusters existentes. Eles foram construídos
+para suportar a ferramenta [kubeadm](/docs/reference/setup-tools/kubeadm/), mas podem ser utilizados em outros contextos para usuários que desejam inicializar clusters sem utilizar o `kubeadm`.
+Foram também construídos para funcionar, via políticas RBAC, com o sistema de [Inicialização do Kubelet via TLS](/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/).
+ 
+<!-- body -->
+## Visão geral dos tokens de inicialização
+ 
+Os tokens de inicialização são definidos com um tipo especifico de segredos (`bootstrap.kubernetes.io/token`) que existem no namespace `kube-system`. Estes segredos são então lidos pelo autenticador de inicialização do servidor de API.
```

Comment:
here it would be good to reference the 'segredos' also as _secrets_ (make the association of the secret object)